fixtures/queries/questions.py

```diff
@@ -0,0 +1,36 @@
+from queries import escape_string
+from queries.subjects import get_subject_id
+
+
+def get_question_id(subject_id, content, image, conn):
+    with conn.cursor() as cur:
+        if image == 'NULL':
+            query = """SELECT id FROM questions WHERE content = '{}' AND image_name IS NULL AND subject_id = '{}'""".format(
+                content, subject_id)
+        else:
+            query = """SELECT id FROM questions WHERE content = '{}' AND image_name = {} AND subject_id = '{}'""".format(
+                content, image, subject_id)
+        cur.execute(query)
```

Comment:
That's what the latest migration was about, I decided to change the question content field to text field instead of varchar(255). It works great with the latest version of the fixtures.